docs/source/gbd2017_models/risk_attributable_causes/chronic_kidney_disease/index.rst

```diff
@@ -106,12 +106,12 @@ Vivarium Modeling Strategy
+The aspects of the disease this cause model is designed to simulate is the basic structure of the disease, its sub causes, associated measures (deaths, prevalence, incidence, emr), associated sequelae, and associated disability weights. The aspects of the disease this cause model is not designed to simulate is the disease structure of CKD, related sub causes, and sequelae. This cause model is designed differently, with a transient disease state titled 'With Condition' based on incidence of CKD. From there, the sub causes and sequelae are categorized within either a 'moderate' or 'severe' CKD state. Across the 5 CKD sub causes, some of the associated sequelae will either be grouped into the 'Moderate' or 'Severe' CKD state. The sequelae which map to 'Severe' CKD state include end stage renal disease sequelae and all Stage V CKD sequelae. These sequelae are fatal and include YLLs and YLDs. All other sequelae are included in the 'Moderate' CKD, which are designated as non-fatal only and include only YLDs. The associated sequelae in each state can be found below in the 'State Severity Split Definitions' table.
 
 Vivarium Modeling Strategy for Risk Factor Impaired Kidney Function (IKF) 
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-Based on the GBD 2017 PAF of 1 relationship between IKF and CKD, simulants are initiated in the model with an assigned value of IKF. Based on this, the simulant will be determined as either moderate or severe CKD. In the treatment ramp for intervention scenarios, this assigned IKF value will determine the care that the simulant receives.
+In this model, simulants are initialized through the following process: Simulants will start as 'susceptible', then based on the prevalence of 'With Condition with CKD' (sum of prevalence of all CKD sub cause sequelae), simulants will be assigned a CKD sequelae value and thus CKD disease state (based on CKD sub cause prevalence). Then, each simulant's sequela value will be mapped to a specific impaired kidney function (IKF) category. The model will include a function which will, based on this category, initialize each simulant with a specific reduced kidney function value.
```

Comment:
In this model, simulants are initialized as "susceptible" or "with condition" through the following process:  simulants will be assigned directly to a CKD sequelae ("with condition" state) based on each sequelae prevalence.  Those not assigned to a sequelae will be initialized to the "susceptible" state.  Each sequelae will then be mapped back to the distribution of IKF based on sequelae based severity splits.  The result will be an IKF value for each simulant that is consistent with sub-cause prevalence.

Reply:
> In this model, simulants are initialized as "susceptible" or "with condition" through the following process: simulants will be assigned directly to a CKD sequelae ("with condition" state) based on each sequelae prevalence. Those not assigned to a sequelae will be initialized to the "susceptible" state. Each sequelae will then be mapped back to the distribution of IKF based on sequelae based severity splits. The result will be an IKF value for each simulant that is consistent with sub-cause prevalence.

I've updated this section with your note above. Is it sufficient and ready to be approved to be merged?